test/fortran/test_fortran_spg_refine_cell.f90

```diff
@@ -0,0 +1,33 @@
+program test_fortran_spg_refine_cell
+    use, intrinsic :: iso_c_binding
+    use spglib_f08, only: spg_refine_cell
+    use test_utils
+    implicit none
+
+    write (*, '("test_FCC")')
+    call test_FCC()
+contains
+    !> @brief Find standardaized cell of FCC from its primitive cell
+    !> The output structure of spg_refine_cell may contain more number of atoms.
+    !> Therefore, larger space than that required for input structure has to
+    !> be allocated.
+    subroutine test_FCC() bind(C)
+        real(c_double) :: lattice(3, 3)
+        real(c_double) :: position(3, 4)
+        integer(c_int) :: types(4)
+        integer :: num_atom_bravais
+        integer(c_int) :: num_atom
+        real(c_double) :: symprec
+
+        num_atom = 1
+        lattice(:, :) = reshape([0, 2, 2, 2, 0, 2, 2, 2, 0], [3, 3])
+        ! position and types : 4 times of number of input atoms at maximum
+        position(:, 1) = 0.0
+        types(1) = 0
+        symprec = 1e-5
+
+        num_atom_bravais = spg_refine_cell(lattice, position, types, num_atom, symprec)
+
+        call assert_int(num_atom_bravais, 4)
```

Comment:
Maybe worth making a:
```fortran
interface assert
  module procedure assert_int, assert_...
end interface
```

Reply:
I agree! I am not yet familiar with fortran and just learning right now. Hopefully this will be done at near future.

Reply:
Yeah Fortran is a mess of a language, I can barely get a footing into it without cursing